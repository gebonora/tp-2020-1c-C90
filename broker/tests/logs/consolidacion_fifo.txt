Partición 1: | 0 - 22 | [X] | Size: 22 b | LRU: 21 | COLA: CATCH | ID: 7 |
Partición 2: | 22 - 27 | [L] | Size: 5 b | LRU: 20 |
Partición 3: | 27 - 47 | [X] | Size: 20 b | LRU: 13 | COLA: CATCH | ID: 4 |
Partición 4: | 47 - 63 | [X] | Size: 16 b | LRU: 16 | COLA: CATCH | ID: 5 |
Partición 5: | 63 - 64 | [L] | Size: 1 b | LRU: 15 |
