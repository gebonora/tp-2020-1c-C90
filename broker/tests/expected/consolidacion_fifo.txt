Partición 1: | 0 - 21 | [X] | Size: 22 b | LRU: 7 | COLA: CATCH | ID: 6 |
Partición 2: | 22 - 26 | [L] | Size: 5 b |
Partición 3: | 27 - 46 | [X] | Size: 20 b | LRU: 4 | COLA: CATCH | ID: 4 |
Partición 4: | 47 - 62 | [X] | Size: 16 b | LRU: 5 | COLA: CATCH | ID: 5 |
Partición 5: | 63 - 63 | [L] | Size: 1 b |
